src/main/vcpf4/usbd_conf.h

```diff
@@ -41,7 +41,7 @@
 #define CDC_CMD_EP                      0x82  /* EP2 for CDC commands */
 
 #define HID_IN_EP                       0x83
-#define HID_IN_PACKET                   8
+#define HID_IN_PACKET                   9
 
```

Comment:
Curious about this setting

Reply:
8 bytes for the axes.
1 byte for the buttons.
Since 8 buttons can match one byte, there is no reason for sending each byte for one button, because state of button can be 1 or 0. (1-bit).

Reply:
![image](https://user-images.githubusercontent.com/7397515/91671278-a43d0080-eb25-11ea-8163-8eec9e92733f.png)